tests/failing

```diff
@@ -1,3 +1,4 @@
+tests/simple/error.uplc
```

Comment:
out of curiosity, why is this test listed as a failing test? I see that there is an expected output file and it looks good to me. What happens if this line is removed?